Compat.h

```diff
@@ -9,6 +9,7 @@ in the source distribution for its full text.
 
 #include "config.h" // IWYU pragma: keep
 
+#include <assert.h> // for _Static_assert() and/or static_assert()
```

Comment:
No need to note the included symbols; at most tag for IWYU if the use of the included symbol is optional (likely the case here).

Please rebase the fixup into the appropriate commit.

Reply:
Done. iwyu 0.19 didn't complain for me but I have put in the tag anyway because it will likely change in future as various branches of the compat shims get exercised.